dhall/src/Dhall.hs

```diff
@@ -362,9 +405,9 @@ inputWithSettings settings (Type {..}) txt = do
     let normExpr = Dhall.Core.normalizeWith (view normalizer settings) expr'
 
     case extract normExpr  of
-        Just x  -> return x
-        Nothing -> Control.Exception.throwIO
-                     (InvalidType expected expr')
+        Success x  -> return x
+        Failure e -> Control.Exception.throwIO
+                     e
```

Comment:
The `e` no longer needs to be on its own line

Reply:
Indeed, that's fixed